controller/proxy-injector/webhook.go

```diff
@@ -107,15 +112,18 @@ func (w *Webhook) inject(request *admissionv1beta1.AdmissionRequest) (*admission
 
 	conf := inject.NewResourceConfig(globalConfig, proxyConfig).
 		WithNsAnnotations(nsAnnotations).
-		WithMeta(request.Kind.Kind, request.Namespace, request.Name)
-	patchJSON, reports, err := conf.GetPatch(request.Object.Raw)
+		WithKind(request.Kind.Kind)
+	supportedResource, err := conf.ParseMeta(request.Object.Raw)
```

Comment:
naming nit: if the `bool` returned by `ParseMeta` strictly indicates whether the payload was empty, recommend just calling this `empty` or similar.